code/modules/mob/language.dm

```diff
@@ -230,10 +230,10 @@
 	var/sounds = rand(2, 8)
 	var/i = 0
 	var/newname = ""
-
+	var/list/vox_name_syllables = list("ti","hi","ki","ya","ta","ha","ka","ya","chi","cha","kah")
```

Comment:
Can make this a static list to avoid it being created every time

Reply:
Good idea actually.

Reply:
Is that why it was it was a GLOBAL LIST? I'm not sure what you mean, to be honest! What is a static list?

Reply:
Thanks for explaining on Discord.